packages/firestore/e2e/QuerySnapshot.e2e.js

```diff
@@ -188,18 +188,21 @@ describe('firestore.QuerySnapshot', () => {
     });
 
     it('calls back a function', async () => {
-      const colRef = firebase.firestore().collection(COLLECTION);
-      colRef.add({});
-      colRef.add({});
-      const snapshot = await colRef.limit(2).get();
-      const callback = sinon.spy();
-      snapshot.forEach.should.be.Function();
-      snapshot.forEach(callback);
-      callback.should.be.calledTwice();
-      callback.args[0][0].constructor.name.should.eql('FirestoreDocumentSnapshot');
-      callback.args[0][1].should.be.Number();
-      callback.args[1][0].constructor.name.should.eql('FirestoreDocumentSnapshot');
-      callback.args[1][1].should.be.Number();
+      // FIXME failing in CI
```

Comment:
I think this one might be saved like the following one: https://github.com/invertase/react-native-firebase/pull/4648/commits/2d9b1c4e91fa4fe05947f29995410427d9ae95d4#diff-bb04291aaf2324e8406b55bd8412111b00ca4d9a60b1f32f1a24d1a96bfcc0bcR213

```
      await Utils.spyToBeCalledTwiceAsync(callback, 20000);
```

...because it mostly works, and if we just give it a timeout vs doing a full halt-and-timeout-whole-test-run it will likely work - this seemed to work in local testing